test/acceptance/environments/local.yml

```diff
@@ -4,3 +4,4 @@ host: httpbin.org
 
 # DNS settings
 dns: 8.8.8.8:53
+timeout: "2000"
```

Comment:
Why do you use a string? Why not `timeout: 2000`?

Reply:
Due value.go restriction in CONF tagname.  I've solved it adding a %v verb
`Error: Error casting timeout. strconv.Atoi: parsing "%!s(float64=2000)": invalid syntax`